api/Navigator.json

```diff
@@ -2365,6 +2365,112 @@
           }
         }
       },
+      "unregisterProtocolHandler": {
```

Comment:
Did you mean to also include `unregisterProtocolHandler` in this PR? If so can you update the description?

Reply:
I changed the PR title.